src/applications/mhv/medications/tests/e2e/pages/MedicationsDetailsPage.js

```diff
@@ -82,6 +82,23 @@ class MedicationsDetailsPage {
       .click({ force: true });
   };
 
+  clickMedicationDetailsLink = prescriptionDetails => {
+    cy.intercept(
+      'GET',
+      `/my_health/v1/prescriptions/${
+        prescriptionDetails.data.attributes.prescriptionId
+      }`,
+      prescriptionDetails,
+    ).as('prescriptionDetails');
+    cy.get(
+      `#card-header-${
+        prescriptionDetails.data.attributes.prescriptionId
+      } > [data-testid="medications-history-details-link"]`,
+    )
+      .should('be.visible')
```

Comment:
split out should('be.visible') from .click();  checking for visibility before clicking is probably more stable.

Reply:
yes, fixed that one and ran 150x . Thanks!